etna/lib/commands.rb

```diff
@@ -238,6 +238,58 @@ def execute(email:, task: false, project_name: nil)
         end
       end
     end
+  end
+
+  class Magma
+    include Etna::CommandExecutor
+
+    class Materialize < Etna::Command
+      include WithEtnaClients
+
+      string_flags << "--project-name"
+      string_flags << "--log-file"
+
+      def logger
+        @logger ||= @logger = Etna::Logger.new(@log_file, 0, 1048576)
+      end
+
+      def execute(project_name:, log_file:'/dev/stdout')
```

Comment:
This is awesome! I can see where this might be expanded in the future, since folks might only want `sc_seq` data for example. Not for this PR, thought.